pages/osx/sed.md

```diff
@@ -1,36 +1,32 @@
 # sed
 
 > Edit text in a scriptable manner.
-> More information: <https://ss64.com/osx/sed.html>.
+> More information: <https://manned.org/sed.1>.
 
-- Replace the first occurrence of a string in a file, and print the result:
+- Execute an expression:
 
-`sed 's/{{find}}/{{replace}}/' {{filename}}`
+`sed "{{expression}}"`
```

Comment:
I feel a number of the new examples on this page are too generic? I'm not convinced that the new layout of this page is more useful than the old one - though I do recognise the merits of showing the generic syntax first.

Reply:
I don't know how to escape placeholder character to write AWK code sample for:

```
`awk '{{{ print $1 }}}'`
```

It is useful because it shows difference between sed and awk.

Reply:
Fixed. ;)